swal-forms.js

```diff
@@ -219,7 +219,8 @@
         return inputTag + labelTag
 
         function toHtmlOptions (optionsString, option) {
-          return optionsString + t("<option value='{value}'>{text}</option>", option)
+            option.selected = ((typeof option.selected !== 'undefined') && option.selected) ? ' selected' : ''
```

Comment:
"(typeof option.selected !== 'undefined')" -> is this check necessary? if options.selected is undefined it would evaluate as false, so I this particular check is not needed.